__tests__/app.test.js

```diff
@@ -239,3 +239,43 @@ describe(" POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  const votesIncrement = { inc_votes: 10 };
+  test("PATCH - status: 201, should update votes of article id", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send(votesIncrement)
+      .expect(201)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toMatchObject({
+          body: expect.any(String),
+          article_id: 1,
+          author: "butter_bridge",
+          votes: 110,
+          created_at: expect.any(String),
+        });
+      });
+  });
+  test("PATCH - status: 400, should return 400 bad request when malformed body", () => {
+    const votesIncrement = {};
+    return request(app)
+      .patch("/api/articles/1")
+      .send(votesIncrement)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request!");
+      });
+  });
+  test("PATCH -status:400, should return bad request when body has incorrect data type", () => {
+    const votesIncrement = { inc_votes: "lol" };
+    return request(app)
+      .patch("/api/articles/2")
+      .send(votesIncrement)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request wrong data type!");
+      });
+  });
```

Comment:
Just need to test what happens if the article_id is invalid or valid but non-existent